samples/backups.js

```diff
@@ -0,0 +1,200 @@
+  console.log('The backup is %s bytes in size.', fetchedBackup.sizeBytes);
+
+  // *Time properties have *Date helpers to convert from ITimestamp to PreciseDate
+  console.log(
+    'The backup will auto delete at',
+    fetchedBackup.expireDate.toISOString() // helper for expireTime
+  );
+
+  console.log(
+    'The backup finished being created at',
+    fetchedBackup.endTime // or use the original timestamp directly
+  );
+  // [END bigtable_get_backup]
+
+  // [START bigtable_update_backup]
+  // Extend a backup's life by updating its expiry date.
+  const [updatedBackup] = await cluster.updateBackup(backupID, {
+    expireTime: new Date(Date.now() + 8 * 60 * 60 * 1000), // 8 hours (was 7)
+  });
+  console.log(
+    'The backup will now auto delete at',
+    updatedBackup.expireDate.toISOString()
+  );
+
+  // alternatively use the update method on the backup
+  // `updatedBackup` can be reused since it is a `Backup`
+  const [ownUpdatedBackupMeta] = await updatedBackup.update({
+    expireTime: new Date(Date.now() + 9 * 60 * 60 * 1000), // 9 hours (was 8)
+  });
+  console.log(
+    'The backup will now auto delete at',
+    ownUpdatedBackupMeta.expireDate.toISOString()
+  );
+  // [END bigtable_update_backup]
+
+  // [START bigtable_restore_backup]
+  // Restore a table to an instance from a backup.
+  // Restores do NOT happen on clusters.
+  const restoredTableId = `${tableID}-restored`;
+  console.log(
+    `Restoring backup ${backupID} to new table ${restoredTableId}...`
+  );
+  const [tableRestoreOperation] = await instance.restoreTable(
+    backupID,
+    clusterID,
+    restoredTableId
+  );
+
+  console.log(
+    'Table restoration started, waiting for the table to be fully restored'
+  );
+  const [restoredTable, restoredMeta] = await tableRestoreOperation.promise();
+  console.log('restoredTable:', restoredTable);
+  console.log('restoredMeta:', restoredMeta);
+  // [END bigtable_restore_backup]
+
+  // [START bigtable_delete_backup]
+  console.log('Deleting the backup %s...', backupID);
+  await cluster.deleteBackup(backupID, {
+    gaxOptions: {
+      // deleting a backup takes a good 30-40 seconds
+      // if DEADLINE_EXCEEDED is witnessed, then deleteBackup exceeds the default timeout
+      // this overrides it
+      timeout: 50 * 1000,
+    },
+  });
+  console.log('Deleted backup with ID %s, no response is returned.', backupID);
+
+  // Alternatively, if a backup object is available, use the `delete` helper.
+  const backupPartial = cluster.asBackup({
+    name: backupFromTable.name, // used from another example
+  });
+
+  // While the Backup instance only really has the name defined,
+  // there is enough information to perform a delete operation.
+  // But, this `delete` helper could even be used after a `listBackups` call.
+  await backupPartial.delete({
+    gaxOptions: {
+      timeout: 50 * 1000,
+    },
+  });
+  // [END bigtable_delete_backup]
+}
+
+require('yargs')
```

Comment:
In our modern samples we do not use `yargs`, and instead use the model found in https://github.com/googleapis/nodejs-bigtable/blob/master/samples/quickstart.js